documentation/tutorials/kaggle_beginner_example_classification.ipynb

```diff
@@ -0,0 +1,2105 @@
+{
```

Comment:
<p>In order to generate (and test) our documentation, we have infrastructure that runs notebooks from start-to-finish. For notebooks requiring Kaggle auth, you should assume that <code>~/.kaggle/kaggle.json</code> already exists. Obviously it won't for <em>users</em> running the notebook, so you will need to handle both paths.</p><p> </p><p>I'm not sure if this is the best example we have, but you can see one approach here: https://www.tensorflow.org/hub/tutorials/text_classification_with_tf_hub_on_kaggle. This checks the local FS first, then prompts the user to upload.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/decision-forests/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>